.vscode/fabricmc.net.code-workspace

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Question: What does this do?

Reply:
VS Code shows errors on the cli files unless I opened the folder on another tab.
Not relevant anymore, figured out another option to fix it.